### DISCUSSION TITLE
Poor performance with large amount of beatmaps

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#16634] [Q&A] [answered]
I imported all my maps from stable (every ranked map, around 80k) and noticed some big stutters when searching in song select and also low fps for a few seconds when exiting back to the main menu. 

https://user-images.githubusercontent.com/22381687/151202379-3eb6dd88-375c-4c13-94e5-c1018b89c42f.mp4

logs:
[network.log](https://github.com/ppy/osu/files/7943288/network.log)
[performance.log](https://github.com/ppy/osu/files/7943289/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7943290/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7943291/updater.log)
[database.log](https://github.com/ppy/osu/files/7943292/database.log)

**Comment by peppy** [comment, marked as answer]
this is to be expected for now. performance is an ongoing effort. it should not affect gameplay.

The next release replaces the database backend and fixed some performance issues but not all, so please wait to test that.
